tests/commands/timezone.py

```diff
@@ -196,5 +196,26 @@ def runTest(self):
                                 KickstartDeprecationWarning,
                                 "The option --isUtc will be deprecated in future releases")
 
+
+class F33_TestCase(F32_TestCase):
+    command = "timezone"
+
+    def setUp(self):
+        super().setUp()
+        warnings.simplefilter("error", category=KickstartDeprecationWarning)
+
+    def runTest(self):
+        # As of Fedora 33 the --ntpservers and --nontp options are considered deprecated.
```

Comment:
Shouldn't this also run the F32_TestCase.runTest? Otherwise we lose all those tests.

Reply:
Looking at the other tests cases for timezone, only the FC6 case runs the FC3 case - so I guess the others a re wrong ? Or I'm missing something obvious. :)

In any case, I'll add the `F32_TestCase.runTest` there and we can fix the other tests with separate PR if they are broken. :)

Reply:
Well, and now it makes the F32 test case fail, as it uses `--ntpservers`, which was deprecated. :D

Maybe that's the reason F32 does not call the older tests, as they would blow up in similar manner, as it also deprecates an option ? paging @VladimirSlavik as the author of the F32 test case :)

Why simple "add a new command" can't always be simple ? :D

Reply:
This should be resolved with the latest bunch of fixes.

Reply:
Sorry, missed this :(